src/components/UserDetail/UserDetail.js

```diff
@@ -28,22 +28,22 @@ class UserDetail extends React.Component {
   static manifest = Object.freeze({
     patronGroups: {
       type: 'okapi',
-      path: 'groups?query=(id=!{user.patronGroup})',
+      path: 'groups?query=(id==!{user.patronGroup})',
       records: 'usergroups',
     },
     openLoansCount: {
       type: 'okapi',
-      path: 'circulation/loans?query=(userId=!{user.id} and status.name<>Closed)&limit=1',
+      path: 'circulation/loans?query=(userId==!{user.id} and status.name<>Closed)&limit=1',
     },
     openAccounts: {
       type: 'okapi',
       records: 'accounts',
-      path: 'accounts?query=(userId=!{user.id} and status.name<>Closed)&limit=100',
+      path: 'accounts?query=(userId==!{user.id} and status.name<>Closed)&limit=100',
     },
     openRequests: {
       type: 'okapi',
       throwErrors: false,
-      path: 'circulation/requests?query=(requesterId=!{user.id} and status=Open)&limit=100',
+      path: 'circulation/requests?query=(requesterId==!{user.id} and status==Open)&limit=100',
```

Comment:
It seems we still need single '=' here for the `status`. It means in this context to fetch all the `Open` requests (`Open - Awaiting pickup`, `Open - Not yet filled`, etc)

Reply:
Thank you for catching that @bohdan-suprun! That said, it rekindles my assertion that `status` should be an object with a boolean attribute `isOpen` so that we _can_ do this kind of query efficiently with something like,  
```
      path: 'circulation/requests?query=(requesterId==!{user.id} and status.isOpen==true)&limit=100',
```
As it is currently, we rely on a convention of starting every request-status with `Open - ` or `Closed - ` which feels really really weak to me.

Reply:
the fact that `status` in some modules is an enum and in others is a foreign key to a lookup table where it _can_ have attributes, but of course doesn't, ugh, don't get me started. 🙄